src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -73,7 +73,7 @@ function default_dimensions(output, grid, indices, with_halos)
     indices = Dict(name => validate_indices(indices, locs[name], grid) for name in keys(locs))
 
     if !with_halos
-        indices = Dict(name => restrict_to_interior.(indices[name], locs[name], topo, size(grid))
+        indices = Dict(name => restrict_to_interior.(indices[name], indices[name], locs[name], topo, size(grid))
```

Comment:
@glwagner I did this here as a quick workaround.

But why the `NetCDFOutputWriter` calls `restrict_to_interior`? Isn't this the job of the `construct_output` method to do that?

Reply:
My memory is hazy and I don't work with it much so I could make a mistake but I think `NetCDFOutputWriter` needs to have indices to generate dimensions (unlike JLD2OutputWriter).